code/_helpers/matrices.dm

```diff
@@ -118,4 +118,22 @@
 
 #undef LUMR
 #undef LUMG
-#undef LUMB
\ No newline at end of file
+#undef LUMB
+
+//Matrix math
+//Given 2 matrices mxn and nxp (row major) it multiplies their members and return an mxp matrix
+//Do make sure your lists actually have this many elements
+/proc/multiply_matrices(list/A, list/B, m, n, p)
```

Comment:
Is this used elsewhere?

Reply:
not that I know of

Reply:
check for references, where'd you even get this lol